### PR TITLE
Add missing lang switchers

### PR DESCRIPTION
**Pull request by acekyd**
## What type of PR is this? (check all applicable)

- [x] 📝 Documentation Update

## Description

This PR adds language switchers to wallet and PFI required SDK guides. 

Depends on #1233 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206571002246566
  - https://app.asana.com/0/0/1206571002246562

### CODE REVIEW

**Review by blackgirlbytes — APPROVED** [review]
Approved! We should make a note to add these to shnips instead of just tabs because the lang switchers dont actually do anything here.

**Comment by acekyd** [comment]
I only just realized what you meant @blackgirlbytes. Nice callout.

File: site/docs/tbdex/pfi/required-sdks.mdx [hunk 1]
```diff
@@ -13,6 +13,8 @@ import importJs from '!!raw-loader!@site/snippets/testsuite-javascript/__tests__
 
 # Required SDKs
 
+<LanguageSwitcher languages="JavaScript, Kotlin" />
```
Comment (angiejones) [inline site/docs/tbdex/pfi/required-sdks.mdx:16]: needs to go above the title to be consistent with all the other guides

File: site/docs/tbdex/wallet/required-sdks.mdx [hunk 2]
```diff
@@ -14,6 +14,8 @@ import importJs from '!!raw-loader!@site/snippets/testsuite-javascript/__tests__
 
 # Required SDKs
 
+<LanguageSwitcher languages="JavaScript, Kotlin" />
```
Comment (angiejones) [inline site/docs/tbdex/wallet/required-sdks.mdx:17]: above title

File: site/docs/tbdex/pfi/required-sdks.mdx (hunk 1 above)
Comment (angiejones) [inline site/docs/tbdex/pfi/required-sdks.mdx:16]: these aren't switching the tabs due to the BreadcrumbTabs not being in sync. this is a bigger effort that needs to be done to tie these two components together. up for it?

https://deploy-preview-1232--tbd-website-developer.netlify.app/docs/tbdex/pfi/required-sdks?lang=Kotlin

**Review by angiejones — CHANGES_REQUESTED** [review]

**Comment by angiejones** [comment]
> Approved! We should make a note to add these to shnips instead of just tabs because the lang switchers dont actually do anything here.

This is the task. We don't want to add dropdowns that don't actually work

File: site/docs/tbdex/pfi/required-sdks.mdx (hunk 1 above)
Comment (acekyd) [reply site/docs/tbdex/pfi/required-sdks.mdx:16]: I noticed that when we have the language switcher below the title in the mdx, we no longer need to add a new title or hide-title flag to the frontmatter. @angiejones. Can change it to the other for consistency.

**Comment by acekyd** [comment]
Depends on #1233

**Comment by acekyd** [comment]
Combining with #1233

**Pull request closed without merging**